### ISSUE TITLE
508/a11y: Filters fieldsets have legends that are dissociated from group

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#345]
When browsing the site using a screen reader (VoiceOver on Mac OS with Safari), the project filter groups have radio buttons that are wrapped in `fieldset` and `legend` elements. Unfortunately, the `legend`s are not announced with the radio buttons as intended due to one minor problem: the `legend` is not the first element within the `fieldset`. They are separated by a `div` which causes the break in accessibility. This lack of association can be confusing for users who cannot see the screen to associate the group names (e.g. Year Completed, Status, Project Type, etc.) with the choices offered within the filter.

> The first element inside the `fieldset` must be a `legend` element, which provides a label or description for the group.
- [W3C WCAG 2.1 Technique H71 - Providing a description for groups of form controls using fieldset and legend elements](https://www.w3.org/WAI/WCAG21/Techniques/html/H71)
